MAINTAINERS.md

```diff
@@ -89,9 +89,10 @@ The `git push --force` line here ensures that the original branch gets updated t
 
 ## Review Drafts
 
-As per [Review Drafts](https://whatwg.org/workstream-policy#review-drafts) editors are expected to publish a Review Draft every six months. This is a manual process ([for now](https://github.com/whatwg/sg/issues/74)):
+As per [Review Drafts](https://whatwg.org/workstream-policy#review-drafts), editors are expected to publish a Review Draft every six months. This is a manual process ([for now](https://github.com/whatwg/sg/issues/74)):
 
 1. Run `make review` and follow the instructions.
```

Comment:
Can you link to an example of `make review`, since it doesn't exist in all Makefiles to begin with?

Reply:
See https://github.com/whatwg/url/pull/387 and https://github.com/whatwg/whatwg.org/pull/201. We should merge those (and equivalent PRs for the former for all others standards) before merging this.

Reply:
OK, made https://github.com/whatwg/fullscreen/pull/125.